.github/workflows/ci.yml

```diff
@@ -472,437 +472,1100 @@ jobs:
     needs:
       - prepare-workflow
-      - build-windows-pkgs
+      - build-windows-pkgs-onedir
+      - windows-2016-ci-deps
     uses: ./.github/workflows/test-packages-action.yml
     with:
       distro-slug: windows-2016
+      nox-session: ci-test-onedir
       platform: windows
       arch: amd64
       salt-version: "${{ needs.prepare-workflow.outputs.salt-version }}"
       pkg-type: MSI
       nox-version: 2022.8.7
+      python-version: "3.10"
       cache-prefix: ${{ needs.prepare-workflow.outputs.cache-seed }}|3.10.13
       skip-code-coverage: ${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}
       skip-junit-reports: ${{ github.event_name == 'pull_request' }}
       testing-releases: ${{ needs.prepare-workflow.outputs.testing-releases }}
 
   windows-2019-nsis-pkg-tests:
-    name: Windows 2019 NSIS Package Tests
+    name: Windows 2019
     if: ${{ fromJSON(needs.prepare-workflow.outputs.jobs)['test-pkg'] && fromJSON(needs.prepare-workflow.outputs.runners)['self-hosted'] }}
     needs:
       - prepare-workflow
-      - build-windows-pkgs
+      - build-windows-pkgs-onedir
+      - windows-2019-ci-deps
     uses: ./.github/workflows/test-packages-action.yml
     with:
       distro-slug: windows-2019
+      nox-session: ci-test-onedir
       platform: windows
       arch: amd64
       salt-version: "${{ needs.prepare-workflow.outputs.salt-version }}"
       pkg-type: NSIS
       nox-version: 2022.8.7
+      python-version: "3.10"
       cache-prefix: ${{ needs.prepare-workflow.outputs.cache-seed }}|3.10.13
       skip-code-coverage: ${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}
       skip-junit-reports: ${{ github.event_name == 'pull_request' }}
       testing-releases: ${{ needs.prepare-workflow.outputs.testing-releases }}
 
   windows-2019-msi-pkg-tests:
-    name: Windows 2019 MSI Package Tests
+    name: Windows 2019
     if: ${{ fromJSON(needs.prepare-workflow.outputs.jobs)['test-pkg'] && fromJSON(needs.prepare-workflow.outputs.runners)['self-hosted'] }}
     needs:
       - prepare-workflow
-      - build-windows-pkgs
+      - build-windows-pkgs-onedir
+      - windows-2019-ci-deps
     uses: ./.github/workflows/test-packages-action.yml
     with:
       distro-slug: windows-2019
+      nox-session: ci-test-onedir
       platform: windows
       arch: amd64
       salt-version: "${{ needs.prepare-workflow.outputs.salt-version }}"
       pkg-type: MSI
       nox-version: 2022.8.7
+      python-version: "3.10"
       cache-prefix: ${{ needs.prepare-workflow.outputs.cache-seed }}|3.10.13
       skip-code-coverage: ${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] }}
       skip-junit-reports: ${{ github.event_name == 'pull_request' }}
       testing-releases: ${{ needs.prepare-workflow.outputs.testing-releases }}
 
   windows-2022-nsis-pkg-tests:
-    name: Windows 2022 NSIS Package Tests
```

Comment:
Why are we removing the additional details off of the windows names? I find them to be useful to know which packages are being tested.

Reply:
You still get the details
![image](https://github.com/saltstack/salt/assets/300048/849b48f5-15e4-44d9-af96-164044f7fad6)